pkg/controller/dataimportcron-controller.go

```diff
@@ -218,20 +219,27 @@ func (r *DataImportCronReconciler) update(ctx context.Context, dataImportCron *c
 		now := metav1.Now()
 		dataImportCron.Status.LastExecutionTimestamp = &now
 
-		if dataVolume.Status.Phase == cdiv1.Succeeded {
+		switch dataVolume.Status.Phase {
```

Comment:
There is an ImportFailed phase.  Not sure if/how it is currently used, may want to check to see if it should be handled here

Reply:
Can be added, but I have no idea how to cause the importer pod to fail. When pod is CrashLoopBackOff the DV phase will still be ImportInProgress.

Reply:
Removing the ImportFailed case, as it's a dead code.